app/views/journal_articles/edit_fields/_date_created.html.erb

```diff
@@ -0,0 +1,7 @@
+<% # use date picker %>
```

Comment:
can you state that this is an override file, and list which version of hyrax you're overriding?